app/diagnostics.go

```diff
@@ -458,6 +459,12 @@ func (a *App) trackConfig() {
 		"file_retention_days":     *cfg.DataRetentionSettings.FileRetentionDays,
 		"deletion_job_start_time": *cfg.DataRetentionSettings.DeletionJobStartTime,
 	})
+
+	SendDiagnostic(TRACK_CONFIG_MESSAGE_EXPORT, map[string]interface{}{
+		"enable_message_export":         *utils.Cfg.MessageExportSettings.EnableExport,
+		"daily_run_time":                *utils.Cfg.MessageExportSettings.DailyRunTime,
+		"default_export_from_timestamp": *utils.Cfg.MessageExportSettings.ExportFromTimestamp,
```

Comment:
This last one doesn't seem like the kind of config value we normally send back in the diagnostics to me.